qt-models/divelocationmodel.h

```diff
@@ -17,7 +17,7 @@ class LocationInformationModel : public QAbstractTableModel {
 public:
 	// Common columns, roles and accessor function for all dive-site models.
 	// Thus, different views can connect to different models.
-	enum Columns { REMOVE, NAME, DESCRIPTION, NUM_DIVES, COORDS, NOTES, DIVESITE, TAXONOMY, COLUMNS};
+	enum Columns { REMOVE, NAME, DESCRIPTION, NUM_DIVES, LOCATION, NOTES, DIVESITE, TAXONOMY, COLUMNS};
```

Comment:
There's something weird going on with your merge of the two PRs. This change appears to be in two of the commits... I'm trying to figure out what happened here, but I'm a bit confused staring at what I see in git...
Ah, I see... yeah, this is confusing, but it looks right